Console/Command/AssetCompressShell.php

```diff
@@ -149,9 +149,12 @@ protected function _clearPath($path, $themes, $targets) {
 				}
 			}
 			if (in_array($base, $targets)) {
-				$this->out(' - Deleting ' . $path . $name);
-				unlink($path . $name);
-				continue;
+				$filepath = realpath($path . DS . $name);
```

Comment:
```/home/alex/Webdev/project/app/tmp/cache/asset_compress``` is a real path. For some reason - I didn't debug and went the cheap way of using realpath - it was missing a trailing DS.